src/server/pps/pretty/pretty.go

```diff
@@ -368,6 +368,8 @@ func prettyTransform(transform *ppsclient.Transform) (string, error) {
 // ShorthandInput renders a pps.Input as a short, readable string
 func ShorthandInput(input *ppsclient.Input) string {
 	switch {
+	case input == nil:
+		return ""
```

Comment:
Let's do something a little prettier than an empty string here, maybe we could do `"none"`.